src/App.tsx

```diff
@@ -1,26 +1,127 @@
 /* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useCallback,
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
 import { UserWarning } from './UserWarning';
+import { USER_ID, deletePost, getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import classNames from 'classnames';
+import { ActionType, TodoContext } from './contexts/TodoContext';
+import { Footer } from './components/Footer';
+import { TodoList } from './components/TodoList';
+import { ErrorContext } from './contexts/ErrorContext';
+import { Header } from './components/Header';
 
-const USER_ID = 0;
+export enum FilterType {
+  ALL = 'ALL',
+  ACTIVE = 'ACTIVE',
+  COMPLETED = 'COMPLETED',
+}
 
 export const App: React.FC = () => {
+  const [filter, setFilter] = useState<boolean | null>(null);
```

Comment:
Consider changing filter on enum with status All Completed Active

It a bad idea to use boolean value if you have more that two filter option